python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -25,6 +31,14 @@
 regex = re.compile('[*?\[\]]')
 logger = getLogger()
 
+def escapeWhiteSpace(somePath):
+    """
+    This makes the aruments safe to pass to the command line
+    Args:
+        somePath (str): somePath which may contain whitespaces or special chars
+    """
+    return quote(somePath)
+
```

Comment:
Why not just remove the function completely and replace with `quote` directly in the code?